### ISSUE TITLE
Library never appears installed

### ISSUE THREAD

**Issue by gratefulfrog** [processing/processing-android#300]
In Android mode, on processing3.3 on Ubuntu 16.04.
Install library Ketai using library manager.
It appears in sketchbook:
![image](https://cloud.githubusercontent.com/assets/1917335/23133086/23c662fc-f790-11e6-87fa-e3d046d6abb4.png)

But not in library manager:

This worked in the previous release of processing!

Is there a workaround?

**Comment by codeanticode** [comment]
Not sure what the problem is, I installed Ketai through the CM (Processing 3.3) and it appears under the Contributed Libraries category. But I'm using Mac, will take a look if I can reproduce the problem on Linux.

**Comment by chepecarlos** [comment]
It also gives me the same problem, I have to remove the. In the code directory

**Comment by DeerajTheepshi** [comment]
Yes, this issue exists in linux due to the case sensitive file system of linux. I have attempted a solution to this issue. Should I make a PR to the main PDE repo ([https://github.com/processing/processing](url) on the same ?

**Comment by DeerajTheepshi** [comment]
https://github.com/processing/processing/pull/5844 - Attempt to solve this issue. What do you think @codeanticode ?
